resources/grch38/reference_data.py

```diff
@@ -0,0 +1,50 @@
+from resources.resource_utils import *
+
+# Resources with no versioning needed
+purcell_5k = TableResource(
+    path='gs://gnomad-public/resources/grch38/purcell5k.ht',
+    source_path="gs://gnomad-public/intervals/purcell5k.interval_list"
+)
+
+na12878_giab = MatrixTableResource(
+    path='gs://gnomad-public/truth-sets/hail-0.2/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.mt',
+    source_path='gs://gnomad-public/truth-sets/source/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz'
+)
+
+# Versioned resources: versions should be listed from most recent to oldest
+clinvar = TableResource.versioned({
+    '20181028': TableResource(
```

Comment:
Should this match the date in the HT/VCF file names?

Reply:
duh! thanks!